code/renderers/vue3/src/docs/sourceDecorator.ts

```diff
@@ -13,7 +13,7 @@ import type {
   InterpolationNode,
   TemplateChildNode,
 } from '@vue/compiler-core';
-import { baseParse } from '@vue/compiler-core';
+import { parse } from '@vue/compiler-dom';
```

Comment:
i don't need the entire compiler-dom for this feature,  compiler-core is minimum  ( baseParse ) i don't see why you need dom ?